tests/rbd/rbd_utils.py

```diff
@@ -662,6 +662,32 @@ def migration_action(self, action, dest_spec):
         log.info(f"Starting the {action} migration process")
         return self.exec_cmd(cmd=f"rbd migration {action} {dest_spec}")
 
+    def create_namespace(self, pool_name, namespace_name):
+        """Create namespace for pool
+
+        rbd namespace create --pool <poolname> --namespace <namespace_name>
+
+        Args:
+            pool_name : Name of the pool within the namespace will create
+            namespace_name : name for the namespace to be created
+        """
+        return self.exec_cmd(
+            cmd=f"rbd namespace create --pool {pool_name} --namespace {namespace_name}"
+        )
+
+    def create_image_namespace(self, pool, namespace, image, size):
+        """Create image inside the namespace
+
+        rbd create --pool <pool> --namespace <namespace_name> <image> --size <size>
+
+        Args:
+            pool= name of pool
+            namespace = name of namespace
+            image: name of image
+        """
+        cmd = f"rbd create --pool {pool} --namespace {namespace} {image} --size {size}"
+        self.exec_cmd(cmd=cmd)
```

Comment:
Did we try modifying exiting method to create image ?
IMO, having two different method creates confusion.

images in namespaces to be treated as same as other images just apart from the name

Reply:
Done. used existing method image_create ().